extensions/wikia/CreateNewWiki/tasks/ImportStarterData.php

```diff
@@ -21,8 +21,15 @@ public function check() {
 	}
 
 	public function run() {
-		// I need to pass $this->sDbStarter to CreateWikiLocalJob::changeStarterContributions
-		$starterDatabase = Starters::getStarterByLanguage( $this->taskContext->getLanguage() );
+		$language = $this->taskContext->getLanguage();
+
+		// TODO Clean up in XW-2393
+		if ( (int) $this->taskContext->getVertical() === \WikiFactoryHub::VERTICAL_ID_TV && $language === 'en' ) {
+			$starterDatabase = Starters::STARTER_EN_TV;
+		} else {
+			// I need to pass $this->sDbStarter to CreateWikiLocalJob::changeStarterContributions
+			$starterDatabase = Starters::getStarterByLanguage( $language );
+		}
```

Comment:
The same as above - c&p logic that should be moved to `getStarterByLanguage` method.